firebase-database/src/test/java/com/google/firebase/database/DatabaseRegistrarTest.java

```diff
@@ -41,7 +41,7 @@ public void getComponents_publishesLibVersionComponent() {
 
   private static FirebaseApp appForDatabaseUrl(String url, String name) {
     return FirebaseApp.initializeApp(
-        RuntimeEnvironment.application.getApplicationContext(),
+        getApplicationContext(),
```

Comment:
```suggestion
        ApplicationProvider.getApplicationContext(),
```

Reply:
The public Google style guide only states that static methods should be referenced from a class and not a variable. The Google employee style guide is stricter and enforces this by requiring static methods to be statically imported.